Rakefile

```diff
@@ -76,3 +76,7 @@ desc "Run example"
 task :example do
   ruby "-w -d -Ilib example/server/server.rb"
 end
+
+task :console do
+  exec 'irb -I lib -r ./lib/liquid.rb'
```

Comment:
I like this, but can you make it a separate PR? also I think `-r liquid` would be enough, don't need the `-r lib/...`